source/unified-test-format/unified-test-format.md

```diff
@@ -3555,6 +3569,8 @@ other specs *and* collating spec changes developed in parallel or during the sam
 
 ## Changelog
 
+- 2025-04-07: Add `topologyOpeningEvent` and `topologyClosedEvent` to the unified test format.
```

Comment:
It may be worth calling out that this was added to the 1.20+ schema versions as it was omitted in error; that would provide context for people as to why this didn't introduce a new schema version.